twitter/README.md

```diff
@@ -55,6 +54,88 @@ To run the tests run the command below:
 ./gradlew test
 ```
 
+## Usage
+
+### Reading from Twitter's Streaming API
+To use Twitter Streaming Endpoints as a source in your pipeline you need
+to create a source by calling the `TwitterSources.stream() or
+TwitterSources.timestampedStream()` methods with a `Properties` object
+that includes Twitter credentials, and a supplier function(`SupplierEx`)
+that provides a specific Twitter streaming endpoint.
+
+The Twitter Stream Source provides tweets as JSON formatted raw strings.
+You can extract related fields from tweets by parsing them into a JSON
+objects.
+
+Here's an example job which the Jet pipelines ingest tweets from
+Twitter's Filtered Stream Endpoint that filters the tweets by keywords.
+```java
+Properties credentials = new Properties();
+properties.setProperty("consumerKey", "???"); // OAuth1 Consumer Key
+properties.setProperty("consumerSecret", "???"); // OAuth1 Consumer Secret
+properties.setProperty("token", "???"); // OAuth1 Token
+properties.setProperty("tokenSecret", "???"); // OAuth1 Token Secret
+List<String> terms = new ArrayList<>(Arrays.asList("BTC", "ETH"));
+StreamSource<String> streamSource =
+             TwitterSources.stream(
+                     credentials,
+                     () -> new StatusesFilterEndpoint().trackTerms(terms)
+             );
+Pipeline p = Pipeline.create();
+p.readFrom(streamSource)
+        .withoutTimestamps()
```

Comment:
Can you align chained method calls around the dot ?

Reply:
I aligned them.